docs/docs/club.md

```
@@ -1,54 +1,220 @@
+- Use the provided official FOSS Club logos
+- Follow specified color schemes and design templates
+- Maintain logo integrity without unauthorised modifications
+- Acknowledge FOSS United support in events and communications
 
-### 11. Celebrate Success and Recognize Contributions:
+Design assets are provided here: [FOSS Club Design Assets/Guide](https://www.figma.com/design/dfTCzPNwORsJmJVmB3QtMj/FOSS-Clubs-Design-Guidelines---Community?node-id=1-84&t=4sLA1AGo2uaunSRV-1).
 
-* Acknowledge and celebrate achievements, contributions, and milestones.
-* Foster a sense of pride, belonging, and camaraderie among club members.
+## 3. Funding and Reimbursements
+#### What funding support is available for FOSS Clubs?
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)